modules/nf-neuro/reconst/fodf/main.nf

```diff
@@ -40,7 +40,7 @@ process RECONST_FODF {
     def sh_order = task.ext.sh_order ? "--sh_order " + task.ext.sh_order : ""
     def sh_basis = task.ext.sh_basis ? "--sh_basis " + task.ext.sh_basis : ""
     def set_method = task.ext.method ? task.ext.method : "ssst"
-    def processes = task.ext.processes ? "--processes " + task.ext.processes : ""
+    def processes = task.cpus > 1 ? "--processes " + task.cpus : ""
```

Comment:
What's the purpose of this line? Why not just add `--processes $task.cpus` directly in the script below? If the number of CPUs is 1, then it should still work. Seems like an avoidable conditional statement.

Reply:
Here I think I wanted to do a little check on the value given, to ensure it is not negative or 0. 1 is included since it is the default for all fodf computing scripts. I can remove it, since it is by no means conventionnal, nor a convention to adopt.